src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -705,7 +716,10 @@ private BoundExpression BindTupleExpression(TupleExpressionSyntax node, Diagnost
 
             //PROTOTYPE: should be a wellknown member?
             var ctor = type.UnderlyingTupleType?.InstanceConstructors.FirstOrDefault();
-
+            if ((object)ctor == null)
+            {
+                return BadExpression(node);
```

Comment:
Ok. I marked as prototype for this PR. Vlad said he'll fix the bad constructor case next (and adding well-known member types).